app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge_theme = @project.challenge.theme
```

Comment:
I'm guessing you wanted to declare this as an instance var because it felt not good to reach through objects in this way in the view. That's a reasonable design choice IMO but it's totally fine to do this type of object manipulation in the view because you're just grabbing data rather than manipulating any of it